gravityflow/gravityflowformconnector
# Fix New Entry mapping for Assignee Fields

## Description
Closes [Issue#96](https://github.com/gravityflow/backlog/issues/96)
Mapping a Workflow Assignee Field to another Workflow Assignee Field for a New Entry Workflow Step doesn't send the data value correctly when the Source Form selection is for a User Role. The data value stored for User Role is in the format `role|administrator`. However, the value getting stored on the target form (with the New Entry data mapping) is mismatched, it just stored `administrator`.

## Testing instructions

1. Create a Source form and a Target Form, both having an Assignee Field.
2. Map the Assignee Field of the two forms with a New Entry Workflow Step on Source Form
3. Create an Approval Step for the Target Form with the mapped Assignee FIeld acting as the Workflow Step Assignee.
4. Create an entry for the Source Form, and select a Role on the Assignee Field.
5. Target Form Entry (generated via New Entry step) would process to workflow completion, despite having an Assignee passed.
6. Now switch to this branch and repeat steps 4 and 5. Target Form entry should be processing the workflow correctly.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
